src/renderer/views/SettingsOverlay/index.tsx

```diff
@@ -147,6 +167,27 @@ function SettingsOverlay({ onClose, isOpen }: OverlayProps) {
     []
   );
 
+  const address = useLoginSession()?.address;
+
+  const debugValue = useMemo(
+    () =>
+      [
+        `LAUNCHER_VERSION: ${versionToDisplay}`,
+        address && `Account: ${address.toString()}`,
```

Comment:
```suggestion
        address && `Account: ${address}`,
```

In TypeScript, the embedded variable in the template string will be applied as `.toString()` automatically. 😅

Reply:
Ah, but I prefer using `.toString()` in this case, because while just print `address` doesn't, using `.toString()` automatically attaches `0x` prefix. which is preferred.

Reply:
AFIK, `Account: ${address.toString()}` and `address && `Account: ${address}` will produce same results.

Reply:
I just tested. and welp, you are right. 
I was confused with `address.toHex()` which was the not prefix giving one.

...but while that, I still think `toString()` should be used for just for preventing future developers get confused about this prefix attaching behavior is default or not. what do you think?

Reply:
Basically, I don't think that `.toString()` usage would help with confusion about 0x prefix because this confusion has been related to `.toString()` and `.toHex()` semantics, not about string interpolation.